megamek/src/megamek/client/ui/swing/tileset/HexTileset.java

```diff
@@ -218,7 +218,8 @@
             }
         }
 
-        Image img = bestMatch.getImage(comp, hex.getCoords().hashCode());
+        Random random = new Random(hex.getCoords().hashCode());
+        Image img = bestMatch.getImage(comp, Math.abs(random.nextInt() * random.nextInt()));
```

Comment:
Is this null actually possible here?

Reply:
Only when there are no base images in the tileset or it didnt load, so only when things are very bad. I'd say this can stay as it was (its the bestMatch.getImage() call and used to be like that before)